exporter/tanzuobservabilityexporter/metrics_test.go

```diff
@@ -395,3 +462,136 @@ func copyTags(tags map[string]string) map[string]string {
 	}
 	return tagsCopy
 }
+
+func verifySumConsumer(t *testing.T, errorOnSend bool) {
```

Comment:
This is a large test to comprehend and understand for the three different types of Aggregation Temporality, as well as handling error on send conditions.

Can we break this into 4 individual tests? For example:

- `TestSumConsumerWithDeltaTemporality`
- `TestSumConsumerWithCumulativeTemporality`
- `TestSumConsumerWithUnspecifiedTemporality`
- `TestSumConsumerWhenErrorSending`